package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "iitc-ce",
+  "description": "ingress intel total conversion (IITC-CE)",
+  "keywords": [
+    "iitc",
+    "ingress"
+  ],
+  "main": "core/total-conversion-build.js",
```

Comment:
I'm not sure about this field.
E.g. I see that Leaflet uses built script path for it: https://github.com/Leaflet/Leaflet/blob/234e00bcab0d89319f4c6874f3063ac311ce8cc6/package.json#L33

Reply:
same as `author`, is this usefull ? this is not a package (yet?)